Extension/package.nls.json

```diff
@@ -117,5 +117,8 @@
     "c_cpp.debuggers.processId.anyOf.description": "Optional process id to attach the debugger to. Use \"${command:pickProcess}\" to get a list of local running processes to attach to. Note that some platforms require administrator privileges in order to attach to a process.",
     "c_cpp.debuggers.symbolSearchPath.description": "Semicolon separated list of directories to use to search for symbol (that is, pdb) files. Example: \"c:\\dir1;c:\\dir2\".",
     "c_cpp.debuggers.dumpPath.description": "Optional full path to a dump file for the specified program. Example: \"c:\\temp\\app.dmp\". Defaults to null.",
-    "c_cpp.debuggers.enableDebugHeap.description": "If false, the process will be launched with debug heap disabled. This sets the environment variable '_NO_DEBUG_HEAP' to '1'."
+    "c_cpp.debuggers.enableDebugHeap.description": "If false, the process will be launched with debug heap disabled. This sets the environment variable '_NO_DEBUG_HEAP' to '1'.",
+    "c_cpp.debuggers.symbolLoadInfo.description": "Supports explcit control of symbol loading. The processing of Exceptions lists and symserver entries requires WaitDynamicLoadLib to be true.",
```

Comment:
Spelling of `explicit` is wrong

Reply:
also `The processing of Exceptions lists` should probably be replaced with a reference to symbol load exceptions list.

Reply:
What are the symserver entries? I Don't think thats something we support in `launch.json`